setup.cfg

```diff
@@ -33,7 +33,7 @@ python_requires = >=3.10,<3.13
 setup_requires = setuptools_scm
 include_package_data = True
 install_requires =
-    numpy>=1.23
+    numpy<2.0.0
```

Comment:
Quick question. If before this needed to be `>=1.23`, should it now still need to be `>=1.23` and `<2.0.0`?

Reply:
Yes!  Good catch!  Fixed.